src/Features/LanguageServer/Protocol/CSharpVisualBasicLanguageServerFactory.cs

```diff
@@ -3,37 +3,40 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Generic;
 using System.Composition;
 using System.IO;
-using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.LanguageServer.Handler;
 using Microsoft.CommonLanguageServerProtocol.Framework;
-using Microsoft.CodeAnalysis.ExternalAccess.Razor;
 using StreamJsonRpc;
-using Microsoft.CodeAnalysis.Host;
 
 namespace Microsoft.CodeAnalysis.LanguageServer
 {
     [Export(typeof(ILanguageServerFactory)), Shared]
     internal class CSharpVisualBasicLanguageServerFactory : ILanguageServerFactory
     {
         private readonly AbstractLspServiceProvider _lspServiceProvider;
+        private readonly IEnumerable<Lazy<ICapabilityRegistrationsProvider>> _capabilityRegistrationsProviders;
```

Comment:
ICapabilityRegistrationsProvider is an ILspService (good) and should not be mef imported via normal means.

This shouldn't be passed in here - wherever we use this service it should be retrieved from the lsp service provider (you can use the ILspServiceFactory if you need it in another LSP service)

Reply:
Done